helm/alfresco-content-services/Chart.yaml

```diff
@@ -14,22 +16,27 @@ home: https://www.alfresco.com
 sources:
   - https://github.com/Alfresco/acs-deployment
 dependencies:
-  - name: alfresco-content-common
-    version: 0.1.0
+  - name: alfresco-common
+    version: 0.1.0-A417
```

Comment:
just a note that after merging https://github.com/Alfresco/alfresco-build-tools/pull/120 the incubator charts by default will be created with the SNAPSHOT suffix, so like `0.1.0-SNAPSHOT`